docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -79,4 +79,8 @@ There you can find links to upgrade notes for other versions too.
         +       arguments:
         +           $useInlineEditation: false
         ```
+- introduced `\Shopsys\FrameworkBundle\Model\Order\Item\OrderItemFactory::createProductFromOrderItemData()` which provides more flexibility when `\Shopsys\FrameworkBundle\Model\Order\Item\OrderItem` is extended
```

Comment:
Upgrade notes should be directives of what should user do when upgrading.

Reply:
I would say something like:
- if you are extending `OrderFacade::refreshOrderItemsWithoutTransportAndPayment()` method in your project, you need to update your usage according to ([#1345](https://github.com/shopsys/shopsys/pull/1345/)) where was logic creating `OrderItem` moved to `OrderItemFactory:: createProductOrderItemFromOrderItemData()` method